mkt/constants/applications.py

```diff
@@ -29,7 +29,15 @@ class DEVICE_GAIA(object):
     api_name = 'firefoxos'
 
 
-DEVICE_TYPE_LIST = [DEVICE_DESKTOP, DEVICE_MOBILE, DEVICE_TABLET, DEVICE_GAIA]
+class DEVICE_TV(object):
+    id = 5
+    name = _(u'Firefox OS TV')
+    class_name = 'firefoxos+tv'
+    api_name = 'firefoxos+tv'
```

Comment:
It is a variant of firefoxos though, and would help going towards having sensible dev/device parameters, like we do for android (where "device" is really the form factor, and "dev" the os, more or less)